lit_tests/coloring.swift

```diff
@@ -3,7 +3,7 @@
 enum List<T> {
   case Nil
   // rdar://21927124
-  // CHECK: <attr-builtin>indirect</attr-builtin> <kw>case</kw> <id>Cons</id>(<type>T</type>, <type>List</type>)
+  // CHECK: <kw>indirect</kw> <kw>case</kw> <id>Cons</id>(<type>T</type>, <type>List</type>)
```

Comment:
Not sure these changes are improvements. Could you explain why this happened?

Reply:
Oh, sorry. Accidentally merged with the apple/swift PR that had approval.

https://github.com/apple/swift/pull/58616 changed the classification behavior of `indirect` from identifier to contextual keyword. Thus, the classification of DeclModifier’s 'Name' child (https://github.com/apple/swift/blob/6f05a95eb16e4e8cc1b5fec733ceda9c41018353/utils/gyb_syntax_support/DeclNodes.py#L168) doesn’t kick in. And I think this is an improvement because `SyntaxClassification.attribute` is described in its doc comment as "An attribute starting with an `@`." (https://github.com/apple/swift-syntax/blob/567acb5d660e3f438792aea180711cd1115c1c1b/Sources/SwiftSyntax/gyb_generated/SyntaxClassification.swift#L38). And `indirect` definitely doesn’t start with a `@` 😉